cluster/saltbase/salt/flannel-server/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Network": "192.168.0.0/16",
```

Comment:
Re why 192: because I've seen confusion on slack as to weather it is possible to use an address range other than 10 (don't know why).

Re why /26 and /16, you have a valid point. We could upgrade the master to have a new network, and all the nodes should take notice. This is something I haven't tested. Feels safer to just leave room anyway.